docs/.gitignore

```diff
@@ -1,4 +1,4 @@
-.jekyll-cache/
+.jekyll-resourceCache/
```

Comment:
I don't think that this change, or the next one, is intended…

Reply:
Haha, good point, will revert it.